src/poprox_recommender/test_offline.py

```diff
@@ -36,7 +36,7 @@ def recsys_metric(recommendations, row_index, news_struuid_ID):
     # use the url of Article
     impressions_truth = (
         pd.read_table(
-            f"{src_dir()}/data/test_mind_large/behaviors.tsv",
+            str(src_dir() / "data" / "test_mind_large" / "behaviors.tsv"),
```

Comment:
same remark as below

Reply:
For feedback like this, I highly recommend the suggestion button 😄 
![Screenshot-20240614174233-67x53](https://github.com/CCRI-POPROX/poprox-recommender/assets/885295/132b3a2a-b098-42e0-8001-351e340f5077)